dbus/methods.go

```diff
@@ -598,3 +598,43 @@ func unitPath(name string) dbus.ObjectPath {
 func unitName(dpath dbus.ObjectPath) string {
 	return pathBusUnescape(path.Base(string(dpath)))
 }
+
+type JobStatus struct {
+	Id       int
+	Unit     string
+	JobType  string
+	Status   string
+	JobPath  string
+	UnitPath string
+}
+
+// ListJobs returns an array with all currently queued jobs
+func (c *Conn) ListJobs() ([]JobStatus, error) {
+	return c.listJobsInternal(c.sysobj.Call("org.freedesktop.systemd1.Manager.ListJobs", 0).Store)
+}
+
+func (c *Conn) listJobsInternal(f storeFunc) ([]JobStatus, error) {
```

Comment:
I think you copied this function-passing approach from other list methods in this module. I haven't touched that code in a while so I don't remember all the implications around that, but do you think it would be possible to avoid doing that here and just directly call the relevant method? I don't think there is much need for being more generic here.

Reply:
Done